src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $white-color;
+  padding-top: 32px;
+  font-size: 10px;
+
+  transition: all 300ms linear;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-inline: 19px;
+  }
+
+  &__content {
+    padding: 40px 16px 16px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+
+    transition: all 300ms linear;
+  }
+
+  &__code {
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $secondary-color;
```

Comment:
The color is set to `$secondary-color`. Ensure that `$secondary-color` is defined in your variables file. If it's not defined, this will cause an error.